Task2/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        { 
+            Console.WriteLine("Введите число 1.");
+            double digit1 = 0;
+            digit1=double.Parse(Console.ReadLine());
+            Console.WriteLine("Введите число 2.");
+            double digit2 = 0;
+            digit2 = double.Parse(Console.ReadLine());
+            Console.WriteLine("Введите число 3.");
+            double digit3 = 0;
+            digit3 = double.Parse(Console.ReadLine());
+
+            double AlgebrAvrg = (digit1 + digit2 + digit3) / 3;
+            double GeomAvrg = Math.Pow(digit1 * digit2 * digit3, (1.0/3));
```

Comment:
Локальные переменные в коде C# принято называть с маленькой буквы.
```suggestion
            double algebrAvrg = (digit1 + digit2 + digit3) / 3;
            double geomAvrg = Math.Pow(digit1 * digit2 * digit3, (1.0/3));
```